common/src/config.rs

```diff
@@ -0,0 +1,125 @@
+use serde::Deserialize;
+use serde_json_core::de;
+
+#[derive(Deserialize)]
+pub struct BootloaderConfigFile {
+    /// Configuration for the frame buffer that can be used by the kernel to display pixels
+    /// on the screen.
+    #[serde(default)]
+    pub frame_buffer: FrameBuffer,
+
+    /// Configuration for changing the level of the filter of the messages that are shown in the
+    /// screen when booting. The default is 'Trace'.
+    #[serde(default)]
+    pub log_level: LevelFilter,
+
+    /// Whether the bootloader should print log messages to the framebuffer when booting.
+    ///
+    /// Enabled by default.
+    #[serde(default)]
+    pub frame_buffer_logger_status: LoggerStatus,
+
+    /// Whether the bootloader should print log messages to the serial port when booting.
+    ///
+    /// Enabled by default.
+    #[serde(default)]
+    pub serial_logger_status: LoggerStatus,
```

Comment:
Could we make these fields `bool`s? I don't like the "Enable"/"Disable" in the config file. I would prefer simple `serial_logging`/`frame_buffer_logging` flags.